packages/sveltekit/src/server/sdk.ts

```diff
@@ -11,9 +12,15 @@ import { applySdkMetadata } from '../common/metadata';
 export function init(options: NodeOptions): void {
   applySdkMetadata(options, ['sveltekit', 'node']);
 
+  addServerIntegrations(options);
+
   initNodeSdk(options);
 
   configureScope(scope => {
     scope.setTag('runtime', 'node');
   });
 }
+
+function addServerIntegrations(options: NodeOptions): void {
```

Comment:
m: before adding this integration, don't we need to check for `hasTracingEnabled`? 

At least that's how we do it on the client-side

Reply:
We don't need to worry about bundle size + the integration should gracefully handle when tracing is not enabled, so I think the answer is no.

We have this logic in next.js because the `http` integration by default sets tracing to be false - you have to opt-in to it (which is what the nextjs SDK adds for you).

Reply:
I see, thanks for clarifying!